etsin_finder/frontend/js/components/qvain/datasets/table.jsx

```diff
@@ -192,10 +192,10 @@ class DatasetTable extends Component {
     return (
       <Fragment>
         <SearchField>
-          <Translate className="visuallyhidden" htmlFor="datasetSearchInput" component={inputLabel} content="qvain.datasets.search" />
+          <Translate className="visuallyhidden" htmlFor="datasetSearchInput" component={inputLabel} content="qvain.datasets.search.hidden" />
           <SearchInput
             id="datasetSearchInput"
-            placeholder="Enter name of dataset"
+            placeholder={translate('qvain.datasets.search.placeholder')}
```

Comment:
Translations using the `translate` function directly don't update automatically when you switch languages. The old translation remains until something else triggers a re-render. When possible, it's better to use the `Translate` component that knows to re-render when the language changes. You can have translated attributes (such as placeholder) by using the `attributes` prop.
```jsx
          <Translate
            component={SearchInput}
            id="datasetSearchInput"
            attributes={{ placeholder: 'qvain.datasets.search.placeholder' }}
```